Riot/Modules/MatrixKit/Utils/EventFormatter/HTMLFormatter.swift

```diff
@@ -47,7 +47,6 @@ class HTMLFormatter: NSObject {
 
         var options: [AnyHashable: Any] = [
             DTUseiOS6Attributes: true,
-            DTDefaultFontDescriptor: font.fontDescriptor,
```

Comment:
I think we should revert was has being done with DTCoreText 1.6.26.
Looking to older commits I see:
```swift
            DTDefaultFontFamily: font.familyName,
            DTDefaultFontName: font.fontName,
            DTDefaultFontSize: font.pointSize,
```
Does it make sense?

Reply:
👍 I just did the same thing to fix the unit tests.